notebooks/quantum-machine-learning/kernel.ipynb

```diff
@@ -746,7 +746,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Thanks for your PR @MarcoBarroca ! Could you remove these changes that are not related with the content? 🙏